### PR TITLE
Add Maintenance Mode and Impersonate Sudo Docs

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds pages for maintenance mode and impersonate other accounts using sudo.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/manage/sudo/impersonate.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ - Your appchain's Sudo account connected to your appchain's Polkadot.js Apps. You can refer to the [Managing Sudo guide](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank} for instructions on injecting your Sudo account into Polkadot.js Apps
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/impersonate/impersonate-1.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Using Sudo As {: #using-sudo-as }
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:31]: I don't use the Sudo pallet, and so the first time I read this, I thought the title got chopped off or something but then I kept reading and realized quickly that wasn't the case. However, I think the name should be changed to maybe "Using the Sudo As Method" or something

File: builders/manage/sudo/impersonate.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+7. Specify the number of tokens to send
+8. Press **SubmitSudo** and confirm the resulting pop-up
+
+![Make a sudo as call](/images/builders/manage/sudo/impersonate/impersonate-2.webp)
+
+The other account had a starting balance of `1,000` tokens prior to the call, and has subsequently dropped to `995` as expected. 
+
+![Check balances on Polkadot.js Apps](/images/builders/manage/sudo/impersonate/impersonate-3.webp)
+
+## Using Sudo and Dispatch As Utility {: #using-sudo-and-dispatch-as-utility }
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:52]:
```suggestion
## Using Sudo and the Dispatch As Utility {: #using-sudo-and-dispatch-as-utility }
```

File: builders/manage/sudo/impersonate.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+The following section will demonstrate using sudo to dispatch calls from an arbitrary origin. When submitting a call in this manner, the runtime will first authenticate the sudo key and then dispatch the call using the `utility` pallet and the `dispatchAs` function, allowing the transaction's origin to be exactly what you'd like.
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:54]: couple cases where Sudo isn't capitalized in this para. please double-check rest of the file

**Review by eshaben — COMMENTED** [review]
Index page icons look great! i ran one of the pages through grammarly quick and saw there were some recommendations on removing words here and there for clarity. if you haven't already, please run these pages through grammarly 🙂

File: builders/manage/sudo/impersonate.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/impersonate/impersonate-1.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Using Sudo As {: #using-sudo-as }
+
+As you know, the Sudo account can perform privileged functions, including impersonating other accounts. When submitting a call via `sudoAs`, the runtime will first authenticate the Sudo key and then dispatch the desired function call with the `Signed` origin from a given account. In the following example, we'll use `sudoAs` to send some tokens to another account. While the result is similar to using sudo with a `forceBalanceTransfer` call, the following example uses a regular balance transfer call where the origin is the sender's account rather than the sudo account.
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:33]: avoid "we'll"

**Comment by themacexpert** [comment]
> Index page icons look great! i ran one of the pages through grammarly quick and saw there were some recommendations on removing words here and there for clarity. if you haven't already, please run these pages through grammarly 🙂

Ty! Looks like I forgot to run grammarly on the impersonate page and missed suggestions on it from the maintenance page too. Fixed!

File: builders/manage/sudo/maintenance.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+![Enable maintenance mode](/images/builders/manage/sudo/maintenance/maintenance-2.webp)
+
+To verify that maintenance mode has been enabled, you can check the **Explorer** section under the **Network** tab and review the recent events. 
+
+![Check maintenance mode is enabled](/images/builders/manage/sudo/maintenance/maintenance-3.webp)
+
+## Exiting Maintenance Mode {: #exiting-maintenance-mode }
+
+To exit maintenance mode and return your appchain to normal operation, navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```
Comment (eshaben) [inline builders/manage/sudo/maintenance.md:49]:
```suggestion
To exit maintenance mode and return your appchain to normal operation, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```

File: builders/manage/sudo/impersonate.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    It's critical to protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Using the Sudo As Method {: #using-the-sudo-as-method }
+
+As you know, the Sudo account can perform privileged functions, including impersonating other accounts. When submitting a call via `sudoAs`, the runtime will first authenticate the Sudo key and then dispatch the desired function call with the `Signed` origin from a given account. In the following example, the `sudoAs` method will orchestrate sending some tokens to another account. While the result is similar to using Sudo with a `forceBalanceTransfer` call, the following example uses a regular balance transfer call where the origin is the sender's account rather than the Sudo account. 
+
+To make a `sudoAs` call to impersonate another account, navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, then you have not associated the Sudo account with Polkadot.js Apps. Ensure that your Sudo account is injected by your wallet and connected to Polkadot.js Apps. Then, take the following steps:
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:35]:
```suggestion
To make a `sudoAs` call to impersonate another account, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, then you have not associated the Sudo account with Polkadot.js Apps. Ensure that your Sudo account is injected by your wallet and connected to Polkadot.js Apps. Then, take the following steps:
```

File: builders/manage/sudo/impersonate.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+The other account had a starting balance of `1,000` tokens before the call, and subsequently dropped to `995` as expected. 
+
+![Check balances on Polkadot.js Apps](/images/builders/manage/sudo/impersonate/impersonate-3.webp)
+
+## Using Sudo and the Dispatch As Utility {: #using-sudo-and-the-dispatch-as-utility }
+
+The following section will demonstrate using Sudo to dispatch calls from an arbitrary origin. When submitting a call in this manner, the runtime will first authenticate the Sudo key and then dispatch the call using the `utility` pallet and the `dispatchAs` function, allowing the transaction's origin to be exactly what you'd like.  
+
+To do so, navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Ensure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```
Comment (eshaben) [inline builders/manage/sudo/impersonate.md:56]:
```suggestion
To do so, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Ensure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```

File: builders/manage/sudo/maintenance.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network/){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/sudo/maintenance/maintenance-1.webp)
+
+!!! warning
+    It's critical to protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Enabling Maintenance Mode {: #enabling-maintenance-mode }
+
+As you know, the Sudo account can perform privileged functions, such as enabling and disabling maintenance mode. To enter maintenance mode and stop regular transaction processing, navigate to the **Developer** Tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```
Comment (eshaben) [inline builders/manage/sudo/maintenance.md:35]:
```suggestion
As you know, the Sudo account can perform privileged functions, such as enabling and disabling maintenance mode. To enter maintenance mode and stop regular transaction processing, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```

**Review by eshaben — APPROVED** [review]
Oops, I just missed this minor capitalization fix. But other than that LGTM! Great job!

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**